st3/sublime_lib/region_manager.py

```diff
@@ -15,6 +15,15 @@ def _coalesce(*values: Optional[T]) -> T:
 
 
 class RegionManager:
+    """A manager for regions in a given :class:`sublime.View` with the same `key`.
+
+    If `key` is not given,
+    a unique identifier will be used.
+
+    If the `scope`, `icon`, and `flags` args are given,
+    then they will be used as default values for :meth:`set`.
+    """
```

Comment:
We need some `versionadded` directive here, right? And include it in `index.rst` ofc.

Reply:
Done.